.gitignore

```diff
@@ -7,15 +7,13 @@ foreman-installer*
 puppet-*
 boxes.yaml
 .idea
-scripts
 repo
-Gemfile.lock
+vagrant/Gemfile.lock
 *.retry
 *.pyc
 user_playbooks
 playbooks/galaxy_roles
 requirements.yml
-.tmp_boxes
```

Comment:
I noticed we remove this and don't re-add it anywhere. Are we no longer using it?

Reply:
This was what we used before ```boxes.d/80-tmp-*``` and is indeed no longer used.